styles/globals.css

```diff
@@ -10,4 +10,6 @@
 .remove-scrollbar {
 -ms-overflow-style: none;  /* IE and Edge */
 scrollbar-width: none;  /* Firefox */
-}
\ No newline at end of file
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Grape+Nuts&family=Patua+One&family=Titillium+Web:ital,wght@0,200;0,400;0,700;1,400&display=swap');
```

Comment:
this commit is from the other PR maybe? 

Make sure to merge that one first, then rebase this PR, then merge it 👍🏽